cli/bench/http/node_http_read_headers.js

```diff
@@ -0,0 +1,10 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+const http = require("http");
+const port = process.argv[2] || "4544";
+console.log("port", port);
+http
+  .Server((req, res) => {
+    const bar = req.headers["foo"];
+    res.end(bar);
+  })
```

Comment:
Is this slower than Deno? I think reading multiple headers (like 10) might make it more obvious.